nengo/cache.py

```diff
@@ -284,16 +285,21 @@ def _check_legacy_file(self):
         legacy_file = os.path.join(self.cache_dir, self._LEGACY)
         if os.path.exists(legacy_file):
             with open(legacy_file, 'r') as lf:
-                version = int(lf.read().strip())
+                text = lf.read()
+            try:
+                lv, pp = tuple(int(x.strip()) for x in text.split('.'))
```

Comment:
The legacy version should always be an int, I would be more worried about the pickle version.